utils/wasm-builder/src/cargo_command.rs

```diff
@@ -36,9 +36,8 @@ pub struct CargoCommand {
     paths_to_remap: Vec<(PathBuf, &'static str)>,
 }
 
-impl CargoCommand {
-    /// Create a new `CargoCommand`.
-    pub fn new() -> Self {
+impl Default for CargoCommand {
+    fn default() -> CargoCommand {
```

Comment:
imho new seemed better this it feels to much calculations and allocations for default impl

Reply:
A `Default` impl will be asked to replace the function `new` by `clippy` once making `CargoCommand` public from the library, just kept `fn new` and introduced Default with 

```rust
impl Default for CargoCommand {
  fn default() -> Self {
     Self::new()
  }
}
```

Reply:
[7b7356e](https://github.com/gear-tech/gear/pull/3835/commits/7b7356e162d79149896c7265573f1373037e933e)